client/beefy/rpc/src/lib.rs

```diff
@@ -136,10 +137,10 @@ where
 	Block: BlockT,
 {
 	fn subscribe_justifications(&self, pending: PendingSubscription) {
-		let stream = self
-			.signed_commitment_stream
-			.subscribe()
-			.map(|sc| notification::EncodedSignedCommitment::new::<Block>(sc));
+		let stream = self.signed_commitment_stream.subscribe().map(|vfp| match vfp {
+			VersionedFinalityProof::V1(sc) =>
+				notification::EncodedSignedCommitment::new::<Block>(sc),
+		});
```

Comment:
I would actually change the exposed RPCs too. User facing API should support versioning.

Reply:
I'll change the exposed RPCs now.